s3/src/main/scala/akka/stream/alpakka/s3/model.scala

```diff
@@ -114,6 +115,244 @@ object MultipartUploadResult {
   )
 }
 
+final class AWSIdentity private (val id: String, val displayName: String) {
```

Comment:
Not sure if you have better suggestions for `AWSIdentity`, basically  this same datastructure is used in many places where you want to identify who or what initiated the request and owns the entity in question

Reply:
Works for me.